src/mono/mono.proj

```diff
@@ -1,5 +1,4 @@
-<Project DefaultTargets="Build">
-  <Import Project="Directory.Build.props" />
+<Project Sdk="Microsoft.Build.Traversal" DefaultTargets="Build">
```

Comment:
You are making this project a traversal even though you don't build any projects here (P2Ps). Why is that so? Can't you usehttps://github.com/microsoft/MSBuildSdks/tree/master/src/NoTargets?

Reply:
I tried that but I would need to set a TargetFramework in order to be able to restore the project. Do we want to se TargetFramework for this project?

Reply:
Not really, I guess Traversal is fine then.

Reply:
Sounds good. Thanks